qa/suites/orch/cephadm/upgrade/3-upgrade/simple.yaml

```diff
@@ -14,6 +14,7 @@ tasks:
       - ceph osd pool create foo
       - rbd pool init foo
       - ceph orch apply iscsi foo u p
+      - ceph orch apply nvmeof foo
```

Comment:
the normal nvmeof deployment test worked, but this upgrade one won't because the upgrade starts from pacific (need to update this to start from quincy actually since we're now in the dev cycle for squid, but that won't help this) and we can't deploy nvmeof on older versions.

Reply:
That sounds reasonable to me.